google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -138,13 +146,18 @@ public void call(
     return new SampleRowKeysCallable(withContext, requestContext);
   }
 
+  /**
+   * Creates a callable chain to handle MutateRow RPCs. The chain will:
+   *
+   * <ul>
+   *   <li>Convert a {@link RowMutation} into a {@link com.google.bigtable.v2.MutateRowRequest}.
+   * </ul>
+   */
   private UnaryCallable<RowMutation, Void> createMutateRowCallable() {
-    return new UnaryCallable<RowMutation, Void>() {
-      @Override
-      public ApiFuture<Void> futureCall(RowMutation request, ApiCallContext context) {
-        throw new UnsupportedOperationException("todo");
-      }
-    };
+    UnaryCallable<MutateRowRequest, MutateRowResponse> withContext =
+        stub.mutateRowCallable().withDefaultCallContext(clientContext.getDefaultCallContext());
+
+    return new MutateRowCallable(withContext, requestContext);
```

Comment:
I think it's best to put the default call context on the outer-most callable.

Reply:
All of the bigtable implementation PRs will have the same issue (to stay consistent with ReadRows). I'll update all of them tomorrow morning

Reply:
Just go ahead and add the same type of comment as you did for #2940 .

Reply:
ReadRows is the only one that needs the special treatment. Since we are planning on removing that need in the future, I'd rather make this consistent with the autogen code and put withDefaultContext as the outermost callable. I just won't be able to do this until tomorrow morning

Reply:
Ahh got it, thanks!